### PR TITLE
Victoria/changelog

### PR DESCRIPTION
**Pull request by victoriabernard92**
Small grammar / Wording nits to the changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,19 +17,19 @@
     [stevenfranks]: https://github.com/stevenfranks
     [pull/782]: https://github.com/cloudflare/wrangler/pull/782
 
-  - **Include/exclude static assets in a Workers Sites project - [gabbifish], [pull/760]**
+  - **Include/exclude static assets in a Worker's Sites project - [gabbifish], [pull/760]**
```
Comment (EverlastingBugstopper) [inline CHANGELOG.md]: image
https://workers.cloudflare.com/sites/

Comment (ashleymichal) [inline CHANGELOG.md]: this was actually correct

Comment (victoriabernard92) [inline CHANGELOG.md]:
```suggestion
  - **Include/exclude static assets in a Workers Sites project - [gabbifish], [pull/760]**
```

**Comment by EverlastingBugstopper** [comment]
closing in favor of https://github.com/cloudflare/wrangler/pull/801/commits/afa2b4ea5f24e866669297b53c0e082dcae62ca8

**Pull request closed without merging**